include/finch/ccsds123b/arrays.h

```diff
@@ -0,0 +1,28 @@
+#ifndef FINCH_ARRAYS_H
+#define FINCH_ARRAYS_H
+
+#include <stdint.h>
+#include <finch/ccsds123b/constants.h>
```

Comment:
This is minor, but it's common to have our files first, and then the C standard library files afterwards. This is because our files are more "important" you could say than the C standard library files